src/field_theory/galois.lean

```diff
@@ -28,6 +28,8 @@ In this file we define Galois extensions as extensions which are both separable
 - `fixed_field_of_fixing_subgroup`: If `E/F` is finite dimensional and Galois
   then `fixed_field (fixing_subgroup K) = K`
 Together, these two result prove the Galois correspondence
+
+- `tfae` : Equivalent characterizations of a Galois extension of finite degree
```

Comment:
```suggestion
- `is_galois.tfae` : Equivalent characterizations of a Galois extension of finite degree
```
Otherwise the docs link to the definition of `_root_.tfae`.